I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/alt/day2.rpy", line 33, in script call
    call displayMenuFreeAfternoon from _call_displayMenuFreeAfternoon_1
  File "game/freeMorning.rpy", line 115, in script call
    call weddingTorielSans
  File "game/wedding/torielSans.rpy", line 28, in script
    gaster "..."
  File "game/characters.rpy", line 25, in gaster_voice
    renpy.voice.play("music/voice/talk_gaster.wav", loop=True)
AttributeError: 'module' object has no attribute 'voice'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/alt/day2.rpy", line 33, in script call
    call displayMenuFreeAfternoon from _call_displayMenuFreeAfternoon_1
  File "game/freeMorning.rpy", line 115, in script call
    call weddingTorielSans
  File "game/wedding/torielSans.rpy", line 28, in script
    gaster "..."
  File "/Applications/renpy-7.3.5-sdk/renpy/ast.py", line 708, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "/Applications/renpy-7.3.5-sdk/renpy/exports.py", line 1345, in say
    who(what, *args, **kwargs)
  File "/Applications/renpy-7.3.5-sdk/renpy/character.py", line 1142, in __call__
    self.do_display(who, what, cb_args=self.cb_args, **display_args)
  File "/Applications/renpy-7.3.5-sdk/renpy/character.py", line 842, in do_display
    **display_args)
  File "/Applications/renpy-7.3.5-sdk/renpy/character.py", line 585, in display_say
    c("show_done", interact=interact, type=type, **cb_args)
  File "game/characters.rpy", line 25, in gaster_voice
    renpy.voice.play("music/voice/talk_gaster.wav", loop=True)
AttributeError: 'module' object has no attribute 'voice'

Darwin-19.5.0-x86_64-i386-64bit
Ren'Py 7.3.5.606
User Interface 0.3.1
Wed Jun 10 20:21:31 2020
